### PR TITLE
Numba

### PR DESCRIPTION
**Pull request by jasonmotha**

### CODE REVIEW

File: empirical/GMM_models/bc_hydro_2016_subduction.py [hunk 1]
```diff
@@ -152,7 +152,10 @@ def bc_hydro_2016_subduction(siteprop, faultprop, im, period):
     sa = np.exp(base + f_mag + f_dep + f_faba + f_site)
     periods_int = np.copy(periods)
     periods_int[0] = 1e-10
-    sa_int = np.interp(np.log(period), np.log(periods_int), sa)
+    if im != "PGA":
```
Comment (sungeunbae) [inline empirical/GMM_models/bc_hydro_2016_subduction.py]: What is the purpose of this? Could you add a comment?

File: empirical/util/empirical_factory.py [hunk 2]
```diff
@@ -36,14 +36,21 @@ def get_models_from_dict(config):
         config = os.path.join(dir, "model_config.yaml")
 
     tect_type_model_dict = yaml.safe_load(open(config))
+    # return list(
```
Comment (sungeunbae) [inline empirical/util/empirical_factory.py]: Any reason for keeping this?

File: empirical/GMM_models/bc_hydro_2016_subduction.py (hunk 1 above)
Comment (jasonmotha) [reply empirical/GMM_models/bc_hydro_2016_subduction.py]: If you have a read through the surrounding lines. Without this check it interpolates every pSA period even if it has it.
When it interpolates, it calculates the log of the period. If the target period is 0, then it takes log of 0. Which results in an error. The interpolation steps has a hack a few lines above to change the 0 into a approximation of 0 (1e-10), which makes this loggable. I think my addition was the 'cleanest' way to 'fix' that bug.

Basically, the BC Hydro empirical model is a bunch of hacks and bad decisions and probably should be rewritten.

File: empirical/util/empirical_factory.py (hunk 2 above)
Comment (jasonmotha) [reply empirical/util/empirical_factory.py]: I'll remove it once we are happy with the database split.
